extensions/BugModal/lib/WebService.pm

```diff
@@ -82,9 +82,10 @@ sub rest_resources {
     ]
 }
 
-sub products {
+sub initial_loader {
     my $user = Bugzilla->user;
-    return { products => _name($user->get_enterable_products) };
+    my @keywords = map { { name => $_->name } } grep { $_->is_active } Bugzilla::Keyword->get_all();
```

Comment:
I'd be happier to see `my @products` here as well, so that +88 below isn't a mix of the two styles, but that's a tiny aesthetic nit and is no obstacle or anything.